tutorials/nlp/question_answering_squad.ipynb

```diff
@@ -0,0 +1,674 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "uRLPr0TnIAHO"
+   },
+   "outputs": [],
+   "source": [
+    "BRANCH = 'candidate'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "o_0K1lsW1dj9"
+   },
+   "outputs": [],
+   "source": [
+    "\"\"\"\n",
+    "You can run either this notebook locally (if you have all the dependencies and a GPU) or on Google Colab.\n",
+    "\n",
+    "Instructions for setting up Colab are as follows:\n",
+    "1. Open a new Python 3 notebook.\n",
+    "2. Import this notebook from GitHub (File -> Upload Notebook -> \"GITHUB\" tab -> copy/paste GitHub URL)\n",
+    "3. Connect to an instance with a GPU (Runtime -> Change runtime type -> select \"GPU\" for hardware accelerator)\n",
+    "4. Run this cell to set up dependencies.\n",
+    "\"\"\"\n",
+    "# If you're using Google Colab and not running locally, run this cell\n",
+    "\n",
+    "# install NeMo\n",
+    "!python -m pip install git+https://github.com/NVIDIA/NeMo.git@{BRANCH}#egg=nemo_toolkit[nlp]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "dzqD2WDFOIN-"
+   },
+   "outputs": [],
+   "source": [
+    "from nemo.utils.exp_manager import exp_manager\n",
+    "from nemo.collections import nlp as nemo_nlp\n",
+    "\n",
+    "import os\n",
+    "import wget \n",
+    "import torch\n",
+    "import pytorch_lightning as pl\n",
+    "from omegaconf import OmegaConf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "daYw_Xll2ZR9"
+   },
+   "source": [
+    "# Task Description\n",
+    "Given a question and a context both in natural language, predict the span within the context with a start and end position which indicates the answer to the question.\n",
+    "For every word in our training dataset we’re going to predict:\n",
+    "- likelihood this word is the start of the span \n",
+    "- likelihood this word is the end of the span \n",
+    "\n",
+    "We are using a pretrained [BERT](https://arxiv.org/pdf/1810.04805.pdf) encoder with 2 span prediction heads for prediction start and end position of the answer. The span predictions are token classfiers consisting of a single linear layer. \n",
+    "The input into the model is going to a concatenation of 2 tokenized sequences: [CLS] query [SEP] context [SEP].\n",
+    "We use [WordPiece](https://arxiv.org/pdf/1609.08144.pdf) Tokenizer and the [Google's BERT vocabulary](https://github.com/google-research/bert). "
```

Comment:
for this tutorial or sentencepiece won't work for this example?

Reply:
Yes it will work, i updated the tutorial with a section that explains how to switch tokenizers

Reply:
I liked how you described it in the tutorial, would be great if you could add it to the NLP docs

Reply:
thanks, good idea! i will add some common nlp documentation especially on tokenizers, but in a new PR